src/names.c

```diff
@@ -48,7 +48,7 @@ netplan_def_type_to_str[NETPLAN_DEF_TYPE_MAX_] = {
     [NETPLAN_DEF_TYPE_BOND] = "bonds",
     [NETPLAN_DEF_TYPE_VLAN] = "vlans",
     [NETPLAN_DEF_TYPE_TUNNEL] = "tunnels",
-    [NETPLAN_DEF_TYPE_PORT] = NULL,
+    [NETPLAN_DEF_TYPE_PORT] = "ovs-ports",
```

Comment:
This is not about the backends/renderers (as described in the commit msg), but about the interface type definition, right?
We do not expose `ovs-ports` to the YAML schema, but this is kind of a special type to carry the relevant data around.

Would you mind changing that name a bit to reflect that this is something special?

```suggestion
    [NETPLAN_DEF_TYPE_PORT] = "_ovs-ports",
```

Reply:
I'll apply the patch, as, even though I'm not very fond of this new syntax, your reasons for adopting it are better than my pure "it looks ugly" approach ;-)